firmware/boron-ins-fsm/test/mocks/Particle.h

```diff
@@ -1,10 +1,26 @@
+/*
```

Comment:
I notice that the other mock files start with `mock_` in the name but this one doesn't. Is there a reason for that?

Reply:
This file was originally present and named Particle.h, and I didn't rename it. I added the mock_ prefix to al the header files so it is clear that they are mocks

Reply:
I found that renaming Particle.h causes the following error:

../src/imDoorSensor.cpp:6:10: fatal error: Particle.h: No such file or directory
    6 | #include "Particle.h"
    9 | #include "mocks/Particle.h"

So I will keep it named Particle.h

Reply:
Cool. Thanks for trying.